presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/PushPartialAggregationThroughJoin.java

```diff
@@ -156,6 +156,7 @@ private AggregationNode replaceAggregationSource(
                 source,
                 aggregation.getAggregations(),
                 ImmutableList.of(groupingSet),
+                ImmutableList.of(),
```

Comment:
you shouldn't create non-streaming partial aggregation for streaming aggregation. This will hurt performance